.actor/actor.json

```
@@ -1,7 +1,7 @@
 {
   "actorSpecification": 1,
   "name": "docling",
-  "version": "0.0",
+  "version": "1.0",
   "environmentVariables": {},
   "dockerFile": "./Dockerfile",
   "input": "./input_schema.json",
```

Based on above ⬆️ discussion:

```suggestion
  "inputSchema": "./input_schema.json",
```